docs/docs/usage/host-functions.md

```diff
@@ -153,9 +158,21 @@ This is because host functions will typically interact with instance state in th
 
 To use the host module, you need to instantiate the host module and fetch the host functions:
 
+<!--
+```java
+// bug in jbang?
```

Comment:
bug reported upstream, the local workaround is not ideal, but good to move forward.